azure/src/test/java/com/microsoft/azure/AzureTests.java

```diff
@@ -76,37 +72,6 @@ public void setup() throws Exception {
     }
 
     /**
-     * Tests the public IP address implementation from an individual service client
-     * @throws Exception 
-     */
-    @Test
-    public void testPublicIpAddressesInGroup() throws Exception {
```

Comment:
sounds reasonable. But this PR hasn't changed the current situation. Let's look at this soon.